code/controllers/subsystem/shuttles.dm

```diff
@@ -47,6 +47,8 @@ var/datum/controller/subsystem/shuttle/SSshuttle
 
 	var/lockdown = FALSE	//disallow transit after nuke goes off
 
+	var/GameTime = 72000 //time before in deciseconds in which the shuttle is auto called. Default is 2 hours.
```

Comment:
Consider changing this to something which identifies it clearly as the time limit.
(maybe without capitals, too)